src/modules/admin/components/Permissions/UserPermissions.tsx

```diff
@@ -40,12 +30,10 @@ const UserPermissions = ({ userAddress, domains, domainId }: Props) => {
 
   return (
     <div className={styles.main}>
-      {/* @TODO restore pending role indicator */}
```

Comment:
I think this `todo` is still useful - the `pending` role indicator refers to the yellow dot that should be visible until a tx is signed / confirmed as a result of an optimistic UI (issue reference: https://github.com/JoinColony/colonyDapp/issues/1708)

Reply:
I'm in favour of just ditching that indicator for now as it is quite hard to do. And getting back to it once we have a more general solution to present the status of transactions (we don't do it anywhere else)